src/main/java/org/clx/library/dto/TransactionDto.java

```diff
@@ -31,4 +34,29 @@ public class TransactionDto {
     private TransactionStatus transactionStatus;
 
     private Date transactionDate;
+
+    public Transaction transactionDtoToTransaction(){
+        Transaction transaction = new Transaction();
+        transaction.setId(id);
+        transaction.setTransactionId(transactionId);
+        transaction.setFineAmount(fineAmount);
+//        transaction.setBook(bookDto.bookDtoToBook());
+        transaction.setIsIssueOperation(isIssueOperation);
+        transaction.setTransactionStatus(transactionStatus);
+        transaction.setTransactionDate(transactionDate);
+        return transaction;
+    }
+
+    public static TransactionDto transactionToTransactionDto(Transaction transaction){
```

Comment:
why do we need to put object return from same class and using static funcationilty? either it can use in a helper class to map the data or bind directly with current object without static and new object.

Reply:
sure we will create helper class for mapping.